src/web/crate_details.rs

```diff
@@ -1222,22 +1259,26 @@ mod tests {
                 ])
                 .create()?;
 
-            let page = kuchikiki::parse_html()
-                .one(env.frontend().get("/crate/foo/latest").send()?.text()?);
-            let link = page
-                .select_first("a.pure-menu-link[href='/crate/foo/0.1.0']")
-                .unwrap();
-
-            assert_eq!(
-                link.as_node()
-                    .as_element()
-                    .unwrap()
-                    .attributes
-                    .borrow()
-                    .get("title")
-                    .unwrap(),
-                "foo-0.1.0 is currently being built"
-            );
+            let response = env.frontend().get("/crate/foo/latest").send()?;
+            // FIXME: temporarily we don't show in-progress releases anywhere, which means we don't
```

Comment:
Same as commented previously.

Reply:
everything about the in-progress releases I would leave in as `FIXME` as a reminder :)